packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -173,6 +178,12 @@ contract FastBridgeV2 is Admin, IFastBridgeV2 {
             rebate
         );
     }
+    
+    /// @inheritdoc IFastBridgeV2
+    function bridgeRelays(bytes32 transactionId) external view returns (bool) {
+        // has this transactionId been relayed?
+        return bridgeRelayDetails[transactionId].blockNumber != 0;
```

Comment:
In my experience there was a very weird bug on one of the chains, where the block timestamp was set to zero. There is no real need in checking `block.number` and `block.timestamp` when saving in `relay()`, but just to be on the safer side of things - let's check `.relayer != address(0)` here? Once the test from the comment above is added, we could be 100% sure this works literally always.